src/main/java/org/mdbenefits/app/config/TransmissionConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package org.mdbenefits.app.config;
+
+import java.util.Map;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "transmission")
+public class TransmissionConfiguration {
+
+    Map<String, String> googleDriveDirectoryId;
```

Comment:
@spokenbird I'm not really sure what value this config class adds. I know I'm not tracking the variables correctly anymore and things still seem to work fine w/o this functioning.  (I think).  Why do I need this class?

Reply:
My understanding is it sets the prefix as valid in application.yaml and enables autocompletion. Without it, the application.yaml won't register the values you add there. Just tried removing it and got:

`Could not resolve placeholder 'BALTIMORE_COUNTY_EMAIL_ADDRESSES' in value "${BALTIMORE_COUNTY_EMAIL_ADDRESSES}"`

Reply:
It was allowing us to have multi keys before a field like foo.bar.thingWeAdd=value, but in this case if it's just `transmission` then I don't think we need it.